artifacts/centosstream/centos-stream_test.go

```diff
@@ -60,8 +59,8 @@ var _ = Describe("CentosStream", func() {
 				Username: "cloud-user",
 			},
 			map[string]string{
-				instancetype.DefaultInstancetypeLabel: "u1.small",
-				instancetype.DefaultPreferenceLabel:   "centos.stream9",
```

Comment:
Why it was changed from `stream9` to `stream8`?

Reply:
Copy paste error, thanks!